models/src_marketo.yml

```diff
@@ -3,10 +3,11 @@ version: 2
 sources:
   - name: marketo
     schema: "{{ var('marketo_schema', 'marketo') }}"
-    database: "{{ var('marketo_database', target.database) }}"
+    database: "{% if target.type != 'spark'%}{{ var('marketo_database', target.database) }}{% endif %}"
```

Comment:
Hmm... I realize I don't know the history of the issues we've had with spark, but for Databricks, my understanding is that it can have up to a three-level namespace (metastore/catalog/schema), though for our configuration we only use two (catalog/schema). The databricks "catalog" maps to "database" in dbt from what I can tell. 

The default "database" is `hive_metastore`, but it could possibly have a different name depending on settings. Also I tested a few difference scenarios of providing database and schema variables, and I found even if no value is given for the database, it does still figures out to use the default. However, I'm thinking we should keep the option to set a database value and keep line 6 as it was since it works? Curious your thoughts! It might also be easier for me to show you. Just let me know!

Reply:
As we chatted about earlier, for incremental models, I have updated `target.type != 'spark'` to  `target.type not in ['spark', 'databricks']`. For the source yml, I have left as-is.